src/main/java/cz/smarteon/loxone/LoxoneEndpoint.java

```diff
@@ -15,17 +15,34 @@
 public final class LoxoneEndpoint {
 
     private final String address;
-    private final int port;
+    private int port;
```

Comment:
what is the value if not set in constructor? What if that unknown value is used somewhere?